client/src/app/site/pages/organization/pages/designs/pages/design-main/components/design-main/design-main.component.ts

```diff
@@ -10,6 +10,6 @@ import { getDesignListSubscriptionConfig } from '../../../../designs.subscriptio
 })
 export class DesignMainComponent extends BaseModelRequestHandlerComponent {
     protected override onShouldCreateModelRequests(): void {
-        this.subscribeTo(getDesignListSubscriptionConfig());
+        this.updateSubscribeTo(getDesignListSubscriptionConfig());
```

Comment:
This is reopening the subscription every time one accesses the design list. We do not want this.